php/WP_CLI/Runner.php

```diff
@@ -595,10 +595,14 @@ public function is_command_disabled( $command ) {
 	/**
 	 * Returns wp-config.php code, skipping the loading of wp-settings.php
 	 *
+	 * @param string $wp_config_file_path config file path
+	 *
 	 * @return string
 	 */
-	public function get_wp_config_code() {
-		$wp_config_path = Utils\locate_wp_config();
+	public function get_wp_config_code( $wp_config_path = '' ) {
+		if ( '' === $wp_config_path ) {
```

Comment:
Using `empty()` here will be safer and also deal with `null`.
```suggestion
		if ( empty( $wp_config_path ) ) {
```